crates/sui-deepbook-indexer/src/lib.rs

```diff
@@ -0,0 +1,398 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
i'd suggest to create a `types.rs` and move things over and make `lib.rs` very lean